modules/ROOT/pages/kubernetes-hardening-guide.adoc

```diff
@@ -0,0 +1,61 @@
+= Kubernetes Hardening Guide
+
+The following guide describes the security challenges associated with setting up and securing a
+Kubernetes cluster based on the NSA (National Security Agency). For complete guidance, see the https://media.defense.gov/2022/Aug/29/2003066362/-1/-1/0/CTR_KUBERNETES_HARDENING_GUIDANCE_1.2_20220829.PDF[Kubernetes Hardening Guide^].
```

Comment:
Maybe based on the guidelines provided NSA?

Based on the NSA just reads a bit weird.